third_party/move/move-compiler-v2/tests/unreachable-code-remover/conditional_loop_unreachable.exp

```diff
@@ -127,19 +87,13 @@ fun m::test($t0: bool, $t1: bool) {
      var $t5: u64 [unused]
      var $t6: u64 [unused]
      var $t7: u64 [unused]
-  0: label L0
-  1: if ($t0) goto 2 else goto 9
-  2: label L2
-  3: if ($t1) goto 4 else goto 7
-  4: label L5
-  5: label L8
-  6: goto 5
-  7: label L6
-  8: goto 11
-  9: label L3
- 10: goto 11
- 11: label L1
- 12: return ()
+  0: if ($t0) goto 1 else goto 5
```

Comment:
I've taken a quick look at these, but please verify stuff under unreachable code remover before merging in as they are new changes due to the change in the pass location.